Maestro/subscriptions.json

```diff
@@ -1151,7 +1151,7 @@
           "GithubRepoOwner": "aspnet",
           "GithubRepoName": "<trigger-repo>",
           "HeadBranch": "<trigger-branch>",
-          "BaseBranch": "master"
+          "BaseBranch": "release/3.0"
```

Comment:
@Pilchie who owns the aspnet/BrowserLink repo these days? I'm not sure whether this change is correct for that repo because it doesn't (yet?) have a 'release/3.0' branch.

@vijayrkn this looks right for aspnet/Scaffolding. Agree?

Reply:
yes. Scaffolding follows the same branching as other .NET Core repos. 

@Jodavis maintains aspnet/BrowserLink